Event-Booking/EventHome_api
# #168926522 Setup Babel and Eslint

#### Description of Task to be completed?
- configure eslint
- configure babel
- install relevant dependencies
- setup basic folder structure
- add husky pre-commit hook

 #### How should this be manually tested?
- clone this branch
- run `npm install` to install the related depencies
- run `npm run dev` to start the development server
- open your browser and navigate to `localhost:5000`
- you should now see a welcome response in json

 #### Any background context you want to provide?
- ensure you have node installed
- active internet connection

 #### What are the relevant pivotal tracker stories?
[#168926522](https://www.pivotaltracker.com/story/show/168926522)

 #### Screenshots (if appropriate)
- none
 #### Questions:
- none
